server/src/datastore/timesheet.repo.ts

```diff
@@ -46,12 +79,29 @@ export class TimesheetRepository extends Repository<Timesheet> {
     });
   }
 
+  proyectTimesheets(projectId: number, from: Date, to: Date): Promise<Timesheet[]> {
+    return this.find({
+      where: {
+        projectId,
+        date: Between(from, to),
+        hours: Not(IsNull())
+      }
+    });
+  }
+
   private getHoursByProject() {
     return this.createQueryBuilder('time')
       .groupBy('time."project_id"')
       .select('SUM(time.hours) as total')
       .addSelect('project_id as "projectId"');
   }
+
+  private getHoursByUser() {
+    return this.createQueryBuilder('time')
+      .groupBy('time."user_id"')
+      .select('SUM(time.hours) as total')
+      .addSelect('"user_id"');
```

Comment:
I think it'd be better to do `.addSelect('user_id as "userId")`, so we stick to the camel case in javascript and snake case for the db